scripts/benchmark_java_arguments.json

```diff
@@ -0,0 +1,19 @@
+{
+    "json-ui": true,
+    "trace": true,
+    "cover": "location",
+    "classpath": ".",
+    "cover-function-only": true,
```

Comment:
Looking at these parameters, I don't think these scripts are well placed in this repository.